### DISCUSSION TITLE
Initialisation of the ROSCO controller with Fl_Mode =2

### DISCUSSION THREAD

**Discussion by MYMahfouz** [NatLabRockies/ROSCO#252] [Q&A]
Hi,

I am using the ROSCO controller v2.7 with a 15Mw floating wind turbine.

When I have `Fl_Mode =2`, I see a big jump in the beginning of the simulation, as shown in the figure bellow. My floater and turbine initial conditions are set properly. What is the cause of this jump?
In some cases the rotor speed goes to negative and fails to recover back. Do I need to change something in my DISCON.IN file to avoid this?

![image](https://github.com/NREL/ROSCO/assets/33868271/ff726361-68d7-41f4-a25d-3beb0bd61635)

Best regards,

**Comment by dzalkind** [comment]
Hi, I believe this was fixed in the latest version (v2.8.0) of ROSCO.

**Reply by MYMahfouz** [reply]
Thanks for your reply. Is there a 64 bit .dll available for the newest version?

**Reply by dzalkind** [reply]
There is now :-)

**Reply by MYMahfouz** [reply]
Thank you. I will try it out.
